odxtools/compumethods/compuscale.py

```diff
@@ -25,7 +25,7 @@ class CompuScale:
         description="<p>fancy description</p>", # optional: provide a description
         lower_limit=Limit(0), # required: lower limit
         upper_limit=Limit(3), # required: upper limit
-        compu_inverse_value=2, # required iff lower_limit != upper_limit
+        compu_inverse_value=2, # required if lower_limit != upper_limit
```

Comment:
small note: in math, the term "iff" means "if and only if", i.e., a bidirectional implication. let's use "if" here because it does not make much of a difference, and I'm not sure whether `compu_inverse_value` is mandatory for intervals containing more than one element...